### ISSUE TITLE
change how tests load live code

### ISSUE THREAD

**Issue by unexre** [unexre/RollTwentyMock#10]
(Previous comments and stackexchange URLs describing the use of `require()();` don't work)

http://stackoverflow.com/a/21424044

Something like this in each test:

```
    var fs = require('fs');
    var vm = require('vm');
    var path = 'path/to/script.js';

    var code = fs.readFileSync(path);
    vm.runInThisContext(code);
```

Note that the path has to be relative to where gulp is run, not where the test file is, which is a bit idiosyncratic.

**Comment by unexre** [comment]
`.spec` files should be next to code.

**Comment by unexre** [comment]
fixed in #14
